python/lsst/ctrl/mpexec/cli/script/report.py

```diff
@@ -24,34 +24,86 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import pprint
 
+import yaml
```

Comment:
Switch the blank line order here. See [PEP8](https://peps.python.org/pep-0008/#imports) for import format.

Reply:
We use isort and black for formatting so I think this is right. `pprint` is part of core python and `yaml` is not so a line separates them.